data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,646 @@ mission "Remnant: Learn Sign Follow Up"
+			
+			branch skip
+				not "Deep: Remnant: Generators: done"
+			`	He pulls out a schematic showing the generator. "The generator on the other hand wasn't all too interesting. Little more than an improved generator of the type you might find down in the Rim. Somewhat interesting that it doesn't produce much energy when compared to something like a fusion reactor given the energy requirement of their engines, but I suppose this people are concerned more with keeping their ships running cool then having excess energy. The one interesting characteristic we discovered though is that the generator seemed to handle the effects of ionization, like something the Hai ion cannons apply, much better than normal. It seemed to be an intentional result of how the generator was designed."`
+			
+			branch skip
+				not "Deep: Remnant: Inhibitor Cannon: done"
+			`	He then pulls out a schematic showing the weapon. "And finally the weapon you brought us. The Inhibitor Cannon is quite the technical marvel, having properties we've never seen before. Imagine a Particle Cannon, but half the size and with more range. Two Inhibitor Cannons would take up about the same amount of space as a single Particle Cannon, yet the Inhibitor Cannons would deal more damage. On top of that, the projectiles of the Inhibitor Cannons have the strange ability to slow down any ship they hit. Ships that we tested them on were unable to turn and accelerate at their normal rates."`
+			
+			label skip
+			`	"Do you think I could get a copy of your research?" you ask.`
+			
+			label suspicion
+			branch cubes
+				has "Deep: Remnant Surveillance: done"
+			branch again2
+				"retrieved information" == 2
+			
+			`	"A copy?" Ivan looks off into space for a moment. "Well, I suppose since you helped me so much, I could make an exception for you. I typically don't share findings with those not on my research team until they're finalized. I've been burned one too many times because of that."`
+			`	Ivan leans over to his desk. "This will be for your eyes only."`
+				goto branches
+			
+			label again2
+			`	"Again?" Ivan asks. He leans over to his desk. "I like that you're so interested in my research. I suppose it only makes sense with you helping me, but typically merchant captains aren't this interested in following up on odd jobs they do."`
```

Comment:
This seems like a prime opportunity for the player to say something about how working for Ivan is far from "just some random odd job." After all, it isn't everyday that a merchant captain gets to help with ground breaking research.

Reply:
Given that this leads directly into the whole "suspicion" branch, I'd rather not make any changes here, as I think it'd ruin the flow of things.

Reply:
OK, fair enough.